src/app/inner.tsx

```diff
@@ -63,11 +62,7 @@ export function Inner({ initialCode }: { initialCode: string | null }) {
                 {!lg.loading && !lg.active && <OutputTabSelectors />}
               </nav>
             </div>
-            <TabContent name="input">
-              <div className="overflow-x-auto bg-neutral-100 outline-2 -outline-offset-2 outline-teal-500 focus-visible:outline dark:bg-neutral-900 dark:outline-teal-300">
-                <Highlight />
-              </div>
-            </TabContent>
+            <CodeEditor initialCode={initialCode ?? defaultCode}/>
```

Comment:
Did you mean to remove `TabContent` here? I think that may be why you can't switch tabs on mobile.

Reply:
Yep that's it :sweat_smile:, thanks